### PR TITLE
Prepare 1.4.2

### PR DESCRIPTION
**Pull request by ametaireau**
* [x] Version dependencies
* [ ] Tag vX.Y.Z
* [ ] Close milestone in Github
* [ ] Publish on Pypi
* [ ] Add entry in Github release page
* [ ] Create next milestone in Github
* [ ] Update version in docs/conf.py
* [ ] Send mail to ML

### CODE REVIEW

**Comment by ametaireau** [comment]
Not ready yet, some more stuff to go in this release haven't been merged yet.

**Comment by ametaireau** [comment]
It's just been merged. Waiting on travis to be green and then we can release.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -6,7 +6,27 @@ This document describes changes between each past release.
 1.4.2 (unreleased)
```
Comment (leplatrem) [inline CHANGELOG.rst]: 1.5 then :)

Comment (ametaireau) [inline CHANGELOG.rst]: Yep :) also, PyFxA have been merged so I'll remove the dependency link here but let the list and everything so it's easier to add later.

**Pull request merged**